draft/2023-10-18-this-week-in-rust.md

```diff
@@ -34,9 +34,9 @@ and just ask the editors to select the category.
 ### Newsletters
 
 ### Project/Tooling Updates
-
+* [Spacedrive Alpha 0.1.1 Released](https://github.com/spacedriveapp/spacedrive/releases/tag/0.1.1)
```

Comment:
Thank you for your submission, @gorentbarak.  Per our submission guidelines, we prefer authored content over changelogs--there is just too much content out there otherwise for the newsletter.